### ISSUE TITLE
The --editor argument doesn't work if you have an argument in your EDITOR env var

### ISSUE THREAD

**Issue by epicserve** [TomAnthony/itermocil#4]
`itermocil --edit sample` didn't work for me because I have the environment variable `EDITOR` set to `subl -w`. When I changed it to just `subl` it worked.

**Comment by TomAnthony** [comment]
I've fixed this now -- if you `brew update` and `brew install TomAnthony/brews/itermocil` you should get v0.1.5 which has the fix.

Thanks for the report, @epicserve. Let me know if you have any other issues. :)

**Comment by epicserve** [comment]
Yep that did it. Only thing I wish it did differently is close the tab you used when you ran `itermocil my_project`. It might be nice to explain the name in the readme. :) Thanks for making this!

**Comment by TomAnthony** [comment]
@epicserve Have you tried using the "--here" flag which will open in the current tab (meaning there is no 'parent' tab to close). I think that should do what you want.

I'm adding tmuxinator support and some other bits, so I'm considering changing the name!

Thanks again for the feedback!

**Comment by epicserve** [comment]
I just tried `--here`, and that did what I wanted. IMHO, that should be the default. Also have you considered making `itermocil -h` the default when there are no arguments. I've seen lots of command-line utils do this (i.e. brew, docker, git, etc.).

**Comment by TomAnthony** [comment]
@epicserve I've made -h the default when there are no arguments (not pushed the code yet).

I'm not sure about `--here` being the default because if people run it in windows with panes it can cause mayhem, and also it isn't the default in teamocil.

However, I can see the appeal and so I think I might introduce an rc file where users can specify this and other defaults they'd like to override. I've added it to my to do list and expect I'll get to it in the next week or two. :)
